static/app/utils/replays/hooks/useReplayOnboarding.tsx

```diff
@@ -21,34 +24,34 @@ function getProjectList(selectedProjects: PageFilters['projects'], projects: Pro
   return selectedProjects.map(id => projectsByProjectId[id]).filter(Boolean);
 }
 
-export function useShouldShowOnboarding() {
+export function useHaveSelectedProjectsSentAnyReplayEvents() {
   const {projects} = useProjects();
   const {selection} = usePageFilters();
 
-  const shouldShowOnboardingPanel = useMemo(() => {
-    const projectList = getProjectList(selection.projects, projects);
-    const hasSentOneReplay = projectList.some(project => project.hasReplays);
-    return !hasSentOneReplay;
```

Comment:
Inverted this name & condition, which means changing the tests and the two callsites (one inside this file)